lib/puppet/pops/model/ast.rb

```diff
@@ -4,19 +4,19 @@ module Puppet
 module Pops
 module Model
 class PopsObject
-  def self._plocation
-    @_plocation ||= (loc = Puppet::Util.path_to_uri("#{__FILE__}"); URI("#{loc}?line=#{__LINE__.to_i - 3}"))
+  def self._pcore_location
```

Comment:
not sure how this ended up being a comment on an outdated change... entered it again on later variant